teamscale-maven-plugin/src/main/java/com/teamscale/tia/maven/TiaMojoBase.java

```diff
@@ -30,19 +35,19 @@
  *     <li>Send test start and end events to the Java agent themselves</li>
  * </ul>
  * <p>
- * To use our JUnit 5 test engine, you must declare it as a dependency of the maven-surefire-plugin and/or the
+ * To use our JUnit 5 impacted-test-engine, you must declare it as a dependency of the maven-surefire-plugin and/or the
  * maven-failsafe-plugin. Example:
  *
  * <pre>{@code
  * <plugin>
  *     <groupId>org.apache.maven.plugins</groupId>
  *     <artifactId>maven-surefire-plugin</artifactId>
- *     <version>3.0.0-M5</version>
+ *     <version>3.0.0</version>
  *     <dependencies>
  *         <dependency>
```

Comment:
In the [docs](https://gitlab.com/cqse/teamscale/teamscale/-/merge_requests/16801/diffs) this dependency is root level/not nested in the surefire plugin configuration.